geonode/base/resource_owner_perm.py

```diff
@@ -0,0 +1,44 @@
+from guardian.shortcuts import get_perms, remove_perm, assign_perm
+from django.conf import settings
+
+
+class ManageResourceOwnerPermissions:
```

Comment:
Why we need a brand new Python file dedicated to this class?

Reply:
will be moved to utils